spec/features/admin_claim_check_spec.rb

```diff
@@ -90,6 +90,16 @@
         expect(page).to have_field("Approve", disabled: true)
         expect(page).to have_content("This claim cannot be approved, the payroll gender is missing")
       end
+
+      context "When a decision has been made" do
+        let!(:claim_missing_payroll_gender_with_decision) { create(:claim, :rejected, payroll_gender: :dont_know) }
+
+        scenario "User is not informed that the claim cannot be approved" do
+          visit admin_claim_path(claim_missing_payroll_gender_with_decision)
+
+          expect(page).to_not have_content("This claim cannot be approved, the payroll gender is missing")
```

Comment:
Minor thing but for specs that check for the absence of something, making the thing you look for as generic and small as possible makes the spec more robust against content changes. So verifying the page does not contain the words `"payroll gender"` would reduce the chance of this spec becoming a false-positive. Or better still, extract the message to the locale file, that way we reduce the duplication here in the specs and in the views and can be confident that content changes won't mean these specs also need to change.